zulipterminal/ui_tools/views.py

```diff
@@ -1230,11 +1230,16 @@ def __init__(self, controller: Any, msg: Message, title: str,
         date_and_time = controller.model.formatted_local_time(
             msg['timestamp'], show_seconds=True, show_year=True
         )
+        view_in_browser_key = ', '.join(map(
```

Comment:
This is rather _keys_?

Reply:
There is only one key for `view_in_browser` i.e. `v`, right?
Is there any particular reason to specify plural?

Reply:
Note that we're join-ing an array. While we do only have one key right now, the list might expand later.